package.json

```diff
@@ -10,38 +10,38 @@
   "license": "(MIT OR Apache-2.0)",
   "main": "",
   "engines": {
-    "node": ">=16.0"
+    "node": ">=18.0"
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "6.5.2",
-    "@fullcalendar/core": "6.1.11",
-    "@fullcalendar/luxon3": "6.1.11",
+    "@fullcalendar/core": "6.1.13",
+    "@fullcalendar/luxon3": "6.1.13",
     "@popperjs/core": "2.11.8",
     "CoolBeans": "0.0.9",
     "awesome-bootstrap-checkbox": "1.0.1",
+    "better-sqlite3": "11.0.0",
```

Comment:
is the sqlite binary part of the dependency?

Reply:
of course.

I did a similar migration (mongo to sqlite) on my jc-backoffice repository a few months ago.